Extensions/PlatformBehavior/PlatformerObjectRuntimeBehavior.h

```diff
@@ -33,6 +33,7 @@ class GD_EXTENSION_API PlatformerObjectRuntimeBehavior
 
   double GetGravity() const { return gravity; };
   double GetMaxFallingSpeed() const { return maxFallingSpeed; };
+  double GetLadderClimbingSpeed() const {return ladderClimbingSpeed};
```

Comment:
Be consistent in the spacings

Reply:
Yup, I am trying to imitate the spaces as they have been done in other files.

Reply:
Ohh! I just noticed the `{return.....}`. Thanks for that